extensions/wikipedia/src/wikipedia.ts

```diff
@@ -51,8 +66,12 @@ export async function findPagesByTitle(search: string, language: string) {
 }
 
 export async function getPageData(title: string, language: string) {
+  const variant = getVariant(language);
+
   const response = await got
-    .get(`${getApiUrl(language)}api/rest_v1/page/summary/${encodeURIComponent(title)}`)
+    .get(`${getApiUrl(language)}api/rest_v1/page/summary/${encodeURIComponent(title)}`, {
+      ...(variant && { headers: { "Accept-Language": variant } }),
+    })
```

Comment:
Suggestion for an alternative way that I find a bit more readable:

```suggestion
    .get(
      `${getApiUrl(language)}api/rest_v1/page/summary/${encodeURIComponent(title)}`,
      variant ? { headers: { "Accept-Language": variant } } : undefined
    )
```